docs/configuration/http_filters/fault_filter.rst

```diff
@@ -66,6 +68,20 @@ upstream_cluster:
   the config are present in the request with the same values (or based on presence if the ``value``
   field is not in the config).
 
+.. _config_http_filters_fault_injection_match_downstream_cluster:
+
+match_downstream_cluster:
```

Comment:
I got hung up on the logic below. I think the original cause was this name. I think we should change this name, and related function to something else: downstream_cluster_specific_settings or something like that.